menu/startup_configuration.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+# T&M Hansson IT AB © - 2020, https://www.hanssonit.se/
+
+# shellcheck disable=2034,2059
+true
+SCRIPT_NAME="Startup Configuration Menu"
+# shellcheck source=lib.sh
+. <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
+
+# Check for errors + debug code and abort if something isn't right
+# 1 = ON
+# 0 = OFF
+DEBUG=0
+debug_mode
+
+# Must be root
+root_check
+
+# Get the correct keyboard layout switch
+if [ "$KEYBOARD_LAYOUT" = "us" ]
+then
+    KEYBOARD_LAYOUT_SWITCH="ON"
+else
+    KEYBOARD_LAYOUT_SWITCH="OFF"
+fi
+
+# Get the correct locate mirror switch
+if [ -z "$REPO" ]
+then
+    REPO=$(apt update -q4 && apt-cache policy | grep http | tail -1 | awk '{print $2}')
+fi
+if [ "$REPO" = "http://se.archive.ubuntu.com/ubuntu" ]
```

Comment:
Actually, in the newer releases we use archive.ubuntu.com.

And also, from a stability perspective, I'd say that it's better to actually *not* run that script. It's just in edge cases where you live like on the other side of the world from the mirrors on slow hardware or a bad WAN. 

As it said in the description before the question, it might not work and we can't guarantee that the mirrors will stay up for the lifetime of the server.

Reply:
> Actually, in the newer releases we use archive.ubuntu.com.

I've tested it with a new VM release (19.0.1) and the server I am checking for was the result.

I only activated this option because you will presented with the question if you really want to run this script which is no by default:
https://github.com/nextcloud/vm/blob/3e5947b755a43c12d40c694cacf25fc9a6cdf382/addons/locate_mirror.sh#L28

But I mean  I can just set this option always to 'No' if you like that more.

Reply:
I've removed that switch and set it to OFF. also I made it the last option